starter-code/pizza.js

```diff
@@ -1 +1,60 @@
 // Write your Pizza Builder JavaScript in this file.
+
+////////////////// Buttons behaviour
+
+// red pepperonnis on/off
+$(".panel .btn-pepperonni").click(function () {
+	$(".pep").toggle();
+	$(".btn-pepperonni").toggleClass("active");
+
+});
+
+
+// mushrooms on/off
+$(".panel .btn-mushrooms").click(function () {
+	$(".mushroom").toggle();
+	$(".btn-mushrooms").toggleClass("active");
+
+});
+
+// peppers on/off
+$(".panel .btn-green-peppers").click(function () {
+	$(".green-pepper").toggle();
+	$(".btn-green-peppers").toggleClass("active");
+});
+
+
+// allowing user to change to white sauce on click
+$(".panel .btn-sauce").click(function () {
+	$(".sauce").toggleClass("sauce-white");
+	$(".btn-sauce").toggleClass("active");
+
+});
+
+// same for crust, switch on button click
+$(".panel .btn-crust").click(function () {
+	$(".crust").toggleClass("crust-gluten-free");
+	$(".btn-crust").toggleClass("active");
+
+});
+
+
+////////////////// Default Pizza Settings 
+
+// setting regular crust and sauce by default (gluten free and white sauce are degeulasse..)
+$(".crust").toggleClass("crust-gluten-free");
+$(".sauce").toggleClass("sauce-white");
+
+// setting the state of crust sauce buttons to off (because there is gluten free or white sauce by default) 
+$(".btn-crust").toggleClass("active");
+$(".btn-sauce").toggleClass("active");
+
+// removing red pepperonis by default (because of vegans)
+$(".pep").toggle();
+$(".btn-pepperonni").toggleClass("active");
+
+
```

Comment:
What you could have done here is wrap everything into a function that is activated when you load the page. 

```
function defaultValues () {
  // Ingredients
  $(".crust").removeClass("crust-gluten-free");
  $(".crust").find(".sauce").removeClass("sauce-white");

  // Buttons state
  $(".btn-pepperonni, .btn-mushrooms, .btn-green-peppers").addClass("active");
  $(".btn-crust, .btn-sauce").removeClass("active");

  // Ingredients list
  $(".panel.price").find("ul").find("li:contains('sauce')").toggle();
  $(".panel.price").find("ul").find("li:contains('crust')").toggle();

  // Total price
  updatePrice();
}
```

Reply:
Then you should call this function at the beginning of your file: 

```
$(document).ready(function() {
  defaultValues();
});
```